File: php/src/Behavioral/Command/DvdCommandNameStarsOff.php
<?

# //DvdCommandNameStarsOff - two of two Concrete Commands

class DvdCommandNameStarsOff extends CommandAbstract
{
	private $dvdName = NULL;

	function __construct(DvdName $dvdName)
	{
		$this->dvdName = $dvdName;
	}

	function execute()
	{
		$this->dvdName->setNameStarsOff();
	}
}

File: csharp/src/Behavioral/Command/DvdCommandNameStarsOff.cs
//DvdCommandNameStarsOff.java - two of two Concrete Commands

namespace Behavioral.Command {


public class DvdCommandNameStarsOff : CommandAbstract {  
   private DvdName dvdName;  
    
   public DvdCommandNameStarsOff(DvdName dvdNameIn) {
       this.dvdName = dvdNameIn;
   }    
   override public void execute() {
       this.dvdName.setNameStarsOff();
   } 
}

}
